File: sample-basic-java/src/main/java/org/dbtools/sample/model/database/main/individual/Individual.java
/*
 * Individual.java
 *
 * Created: 05/07/2016 08:50:53
 */



package org.dbtools.sample.model.database.main.individual;

import android.database.Cursor;
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues;


public class Individual extends IndividualBaseRecord {


    public Individual(Cursor cursor) {
        setContent(cursor);
    }

    public Individual(DBToolsContentValues values) {
        setContent(values);
    }

    public Individual() {
    }


}

File: sample-basic-kotlin/src/main/kotlin/org/dbtools/sample/kotlin/model/database/main/individual/Individual.kt
/*
 * Individual.kt
 *
 * Created: 05/09/2016 09:21:13
 */



package org.dbtools.sample.kotlin.model.database.main.individual

import android.database.Cursor
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues


class Individual : IndividualBaseRecord {


    constructor() {
    }

    constructor(cursor: Cursor) {
        setContent(cursor)
    }

    constructor(values: DBToolsContentValues<*>) {
        setContent(values)
    }


}